components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth2/bean/ScopeTest.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wso2.carbon.identity.oauth2.bean;
+
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import static org.testng.Assert.assertTrue;
+import static org.testng.Assert.assertNotEquals;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotNull;
+
+@PrepareForTest(StringBuilder.class)
+public class ScopeTest {
+    Scope scope;
+    String name = "readClaims";
+    String description = "Test cases to test scope";
+
+    ArrayList<String> bindings = new ArrayList<>(Arrays.asList("scope1", "scope2"));
+
+    @BeforeTest
+    public void setUp() throws IllegalAccessException, InstantiationException {
+        scope = new Scope(name, description);
```

Comment:
scope variable is assigned twice meaning the first assignment is useless.

Reply:
Defined two variables for the two constructors.

Reply:
@ChathurangiShyalika  I only see one at line 32 still. Am I missing something?